app/Makefile

```diff
@@ -22,10 +22,27 @@ endif
 export RUN_USER
 export RUN_UID
 
+##################################################
+# Build
+##################################################
 release-build:
 	docker buildx build \
 		--platform=linux/amd64 \
 		--build-arg RUN_USER=$(RUN_USER) \
 		--build-arg RUN_UID=$(RUN_UID) \
 		$(OPTS) \
 		.
+
+##################################################
+# Local development
+##################################################
+dev: # Run the Next.js local dev server in Docker
+	docker compose --file ../docker-compose.yml up --detach nextjs
```

Comment:
Do we need to specify `../docker-compose.yml` in these commands? We don't have that in the API, I think Docker just knows it is in the directory up?

Reply:
Oh yea, this was a leftover for when I had the separate file for dev. I'll remove this part